diagnostic.md

```diff
@@ -6,70 +6,73 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+esources :books, only: [:index, :show]
+or
+resources :books, except: [:new, :edit]
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+config/routes.rb
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+we are calling the bundler to execute any/all scripts in db/migrate directory
 ```
 
 What is the difference between `rails g scaffold things` and
 `rails g controller things`?
 
 ```md
-// your response here
+you are generateing a model and generating a controller, respectively
+the scaffold will create a model, a controller for that model, as well as default views
```

Comment:
The first command generates routes, a controller and model for foo. The second generates just a foo controller.